pkg/apply/processor/install.go

```diff
@@ -140,17 +140,46 @@ func (c *InstallProcessor) PreProcess(cluster *v2.Cluster) error {
 			imageTypes.Insert(string(v2.AppImage))
 		}
 	}
+	syncMounts := make([]v2.MountImage, 0)
+	imageKeys := sets.NewString()
+	if cluster.Status.Mounts != nil {
+		for _, mot := range cluster.Status.Mounts {
+			if imageKeys.Has(mot.ImageName) {
+				continue
+			}
+			imageKeys.Insert(mot.ImageName)
+			syncMounts = append(syncMounts, mot)
+		}
+	}
+	cluster.Status.Mounts = syncMounts
 	for _, img := range c.NewImages {
+		index, mount := cluster.FindImage(img)
 		var ctrName string
-		mount := cluster.FindImage(img)
 		if mount != nil {
 			if !ForceOverride {
 				continue
 			}
-			logger.Debug("trying to override app %s", img)
-			if err := c.Buildah.Delete(mount.Name); err != nil {
+			allContainers, err := c.Buildah.ListContainers()
```

Comment:
why do we need to remove all the containers that referenced to this image?

Reply:
If there are a large number of registry directories, they will occupy disk space. For example, if the image of kb is 4GB, it will be full if it is executed several times.

Reply:
Would executing `sealos rm/rmi` commands periodically be more appropriate? We should keep as many run records as possible instead of cleaning it up automatically.

Reply:
The contents of all containers are basically the same. There is no need to keep it. Just keep the latest one.